executor/insert_common.go

```diff
@@ -200,6 +201,13 @@ func insertRows(ctx context.Context, base insertCommon) (err error) {
 	if err = e.processSetList(); err != nil {
 		return err
 	}
+	var memTracker *memory.Tracker
+	switch x := base.(type) {
```

Comment:
maybe better add a `default` clause and panic to prevent others pass other type `base` and memTracker be nil,  reported by goland `Niless analyzer` warning~